src/Symfony/Component/Messenger/Exception/HandlerFailedException.php

```diff
@@ -49,4 +49,19 @@ public function getNestedExceptions(): array
     {
         return $this->exceptions;
     }
+
+    public function hasNestedExceptionOfClass(string $exceptionClassName): bool
+    {
+        return 0 < \count($this->getNestedExceptionOfClass($exceptionClassName));
```

Comment:
I'm not sure about this method: it looks like calling for double computation of the logic.

Reply:
Hi ! You mean, I should remove it or you have an idea to improve it ?

Reply:
If we have 20 exceptions, and the first one is `$exceptionClassName`, we will still could the other 19. Right?

Reply:
can you please remove this method?